config/settings/qa.yml

```diff
@@ -1,3 +1,5 @@
+publish_url: https://www.qa.publish-teacher-training-courses.service.gov.uk
```

Comment:
Shall we set it up for paas already? It's https://qa.publish-teacher-training-courses.service.gov.uk

Reply:
done, cheers